ale_linters/llvm/llc.vim

```diff
@@ -0,0 +1,35 @@
+" Author: rhysd <https://rhysd.github.io>
+" Description: Support for checking LLVM IR with llc
+
+call ale#Set('llvm_llc_executable', 'llc')
+
+function! ale_linters#llvm#llc#GetExecutable(buffer) abort
+    return ale#Var(a:buffer, 'llvm_llc_executable')
+endfunction
+
+function! ale_linters#llvm#llc#GetCommand(buffer) abort
+    return ale_linters#llvm#llc#GetExecutable(a:buffer)
```

Comment:
Use `ale#Escape` on the executable here. Otherwise, paths with spaces and other characters won't work.